File: backend/src/main/kotlin/ua/nure/makieiev/ark/model/dto/UserPersonalScheduleDto.kt
package ua.nure.makieiev.ark.model.dto

import java.time.LocalDate
import javax.validation.constraints.NotNull

data class UserPersonalScheduleDto(val id: Long,
                                   @NotNull
                                   val personalScheduleId: Long? = null,
                                   @NotNull
                                   val userLogin: String? = null,
                                   @NotNull
                                   val workDate: LocalDate? = null)

File: frontend/ark-project/src/app/models/dto/UserPersonalScheduleDto.ts
export class UserPersonalScheduleDto {

  personalScheduleId: number;
  userLogin: string;
  workDate: string;

  constructor(personalScheduleId: number, userLogin: string, workDate: string) {
    this.personalScheduleId = personalScheduleId;
    this.userLogin = userLogin;
    this.workDate = workDate;
  }

}
